### PR TITLE
Add new config settings to configure number of oracle test nodes

### PR DESCRIPTION
**Pull request by aleksbykov**
new parameter n_ot_nodes (number oracle test nodes in cluster).
if not set in configuration file, default is 1.

### CODE REVIEW

File: sdcm/sct_config.py [hunk 1]
```diff
@@ -95,6 +95,9 @@ class SCTConfiguration(dict):
         dict(name="n_db_nodes", env="SCT_N_DB_NODES",  type=int_or_list,
              help="Number list of database nodes in multiple data centers."),
 
+        dict(name="n_ot_nodes", env="SCT_N_OT_NODES",  type=int_or_list,
```
Comment (roydahan) [inline sdcm/sct_config.py]: Name is not clear for someone that reads it.
please change it to: n_gemini_test_oracle_db_nodes

Comment (aleksbykov) [reply sdcm/sct_config.py]: Done

File: sdcm/sct_config.py [hunk 2]
```diff
@@ -95,6 +95,9 @@ class SCTConfiguration(dict):
         dict(name="n_db_nodes", env="SCT_N_DB_NODES",  type=int_or_list,
              help="Number list of database nodes in multiple data centers."),
 
+        dict(name="n_gemini_test_oracle_db_nodes", env="SCT_n_gemini_test_oracle_db_nodes",  type=int_or_list,
```
Comment (roydahan) [inline sdcm/sct_config.py]: Shouldn't it be all capital?
SCT_N_GEMINI_TEST...

Comment (aleksbykov) [reply sdcm/sct_config.py]: yep, you right. Fixed

Comment (bentsi) [reply sdcm/sct_config.py]: `SCT_N_TEST_ORACLE_DB_NODES`

File: sdcm/sct_config.py [hunk 3]
```diff
@@ -95,6 +95,9 @@ class SCTConfiguration(dict):
         dict(name="n_db_nodes", env="SCT_N_DB_NODES",  type=int_or_list,
              help="Number list of database nodes in multiple data centers."),
 
+        dict(name="n_gemini_test_oracle_db_nodes", env="SCT_N_GEMINI_TEST_ORACLE_DB_NODES",  type=int_or_list,
```
Comment (bentsi) [inline sdcm/sct_config.py]: make it `SCT_N_TEST_ORACLE_DB_NODES`

File: sdcm/sct_config.py (hunk 2 above)
Comment (roydahan) [reply sdcm/sct_config.py]: We can also remove gemini from the name.

File: sdcm/sct_config.py (hunk 3 above)
Comment (roydahan) [reply sdcm/sct_config.py]: I agree.
(also for the yaml param)

**Pull request merged**